packages/scoutgame-ui/src/components/claim/components/PointsClaimScreen/PointsClaimScreen.tsx

```diff
@@ -90,7 +164,7 @@ export function PointsClaimScreen({
               </Typography>
               <Stack flexDirection='row' alignItems='center' gap={1}>
                 <Typography variant='h4' fontWeight={500}>
-                  {totalUnclaimedPoints}
+                  {totalUnclaimedPoints.toLocaleString()}
```

Comment:
nice catch

Reply:
The unclaimed points are already in human friendly format, so we should be good on this front